forta-staking-parameters/src/agent.js

```diff
@@ -8,6 +8,8 @@ const handleTransaction = async (txEvent) => {
   const txFiltered = txEvent.filterLog(abi, contract);
 
   txFiltered.forEach((tx) => {
+    const from = txEvent.from;
+    const fromLowerCase = from.toLowerCase();
```

Comment:
We don't need to call toLowerCase because the txEvent.from has only lowercase characters